compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -4094,7 +4094,7 @@
   c. Returns true or false"
   [c x]
   ;; (. c (isInstance x))
-  (throw "TODO: port"))
+  (throw "TODO: port instance?"))
```

Comment:
It's fine to change all of these, but it wasn't needed to find your error. For that, I'd just hop into gdb and tell it to stop when an exception is thrown.

```
gdb --args ./build/jank run foo.jank
> catch throw
> run
... wait for an exception to be thrown and gdb will stop ...
> backtrace
... see where you are ...
```

Reply:
Aha, so `(byte 1)` will give `#2  0x00007fffe8df885d in clojure_core_byte_3074_1 ()`.

Very instructive thanks.

Reply:
Not quite. `clojure_core_byte_3074_1` means `clojure.core/byte <gensym count> <arity 1>`.

Reply:
Ok. I pointed it out because AFAICT this is the only frame that gives a hint of which function needs implementing. Is there a way to configure source mapping? e.g.,

```clojure
;;foo.jank
(ns foo)
(defn -main [] (byte 1)) 
```

```cpp
#8  0x00007ffff33850e0 in foo.main_1_0 >>here>> at foo.jank:2 <<here<< ()
```

Reply:
There will be, but it's a *large* feature which involves a lot of LLVM IR work. Perhaps I know the right guy for it, though. I'll reach out to him.